bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -105,9 +105,10 @@ public void textBoxKeyPress(final TextBoxWidget tbwe) {
         _agsPub.subscribe((obs, oldOptions, newOptions) -> updateGuiding());
 
         BagsManager.addBagsStateListener((key, oldStatus, newStatus) -> {
-            if (key.equals(getContextObservation().getNodeKey()))
+            if (Optional.ofNullable(getContextObservation()).map(co -> key.equals(co.getNodeKey())).orElse(false)) {
```

Comment:
This has been something I've been wondering about for awhile: do we want to make a decision moving forward regarding consistency about using our own `Option` or Java 8's `Optional`?